tilelang/tools/layout_visual.py

```
@@ -0,0 +1,64 @@
+                    if layout_id not in self.processed_layouts:
+                        print(f"{key} layout inference:")
+                        print(print_layout_format(layout))
+                        plot_layout(layout, name=f"{key}_layout", formats=self.formats)
+                        self.processed_layouts.add(layout_id)
+
+        self.visit_stmt(op.body)
+
+
+def LayoutVisual():
+
+    def pass_fn(func: tir.PrimFunc, mod, ctx):
```

Ruff (0.14.6)

Unused function argument: `mod`

(ARG001)